src/client/opamAuxCommands.ml

```diff
@@ -303,16 +303,31 @@ let autopin_aux st ?quiet ?(for_view=false) ?recurse ?subpath ?locked
               | _ -> false)
            | None -> false)
           &&
-          (* For `opam show`, we need to check does the opam file changed to
```

Comment:
It worth adding a test for `opam show` that highlights that there is no change in the behaviour